src/components/Ledger/LedgerView/LedgerFunds.js

```diff
@@ -30,42 +21,18 @@ const LedgerFunds = ({ history, funds, currency, mutator, fiscalYears, resources
     return null;
   }, [fiscalYears]);
 
-  useEffect(() => {
-    mutator.groupFundFiscalYears.reset();
-    mutator.groupFundFiscalYears.GET({
-      params: {
-        limit: LIMIT_MAX,
-        query: buildQuery,
-      },
-    });
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [buildQuery]);
-
   const fundFiscalYears = get(resources, ['groupFundFiscalYears', 'records'], []);
 
-  const groupedFundFiscalYears = groupBy(fundFiscalYears, 'fundId');
-
-  const fundsToDisplay = funds.map(fund => {
-    const groupFundFiscalYear = groupedFundFiscalYears[fund.id];
-
-    if (!groupFundFiscalYear) return fund;
-
-    const sum = groupFundFiscalYear.reduce((result, item) => {
-      Object.keys(result).forEach(key => {
-        if (item[key]) result[key] += item[key];
-      });
-
-      return result;
-    }, { available: 0, allocated: 0, unavailable: 0 });
-
-    return { ...sum, ...fund };
-  });
+  const fundsToDisplay = getFundsToDisplay(funds, fundFiscalYears);
 
   return (
-    <ConnectionListing
-      items={fundsToDisplay}
+    <RelatedFunds
+      fundsToDisplay={fundsToDisplay}
       currency={currency}
-      openItem={openFund}
+      parentMutator={mutator}
```

Comment:
that look disturbing, what the reason of adding that `parent*` things? It's an obsolete pattern

Reply:
removed this props and used stripesConnect in common component